src/cljs/rems/administration/create_form.cljs

```diff
@@ -134,18 +134,21 @@
   (into {} (for [language languages]
              [language (trim-when-string (get lstr language ""))])))
 
+(defn- string-is-not-empty [s]
+  (when (string? s)
+    (not (clojure.string/blank? s))))
```

Comment:
1) `(clojure.string/blank? nil)` is `true` so no need for the `when` here
2) you can use `str/blank?` since we have a `(:require [clojure.string :as str])` in this namespace

Reply:
Good point!